packages/terra-clinical-result/src/ClinicalResult.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconModified from 'terra-icon/lib/icon/IconModified';
+import IconComment from 'terra-icon/lib/icon/IconComment';
+import IconUnverified from 'terra-icon/lib/icon/IconDiamond';
+import ResultError from './common/other/_ResultError';
+import NoData from './common/other/_KnownNoData';
+import Observation from './common/observation/_Observation';
+import observationPropShape from './proptypes/observationPropTypes';
+import styles from './ClinicalResult.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Result Object with the clinical result data.
+   */
+  resultData: observationPropShape,
+  /**
+   * Visually hides the unit of measure when presented in a series of side-by-side columns of the same unit.
+   */
+  hideUnit: PropTypes.bool,
+  /**
+   * Whether or not the text should be truncated in display. Restricts clinical result details each to one line.
+   */
+  isTruncated: PropTypes.bool,
+  /**
+   * Override that shows an Error display. Used when there is a known error or problem when retrieving or assembling the clinical result data.
+   */
+  hasResultError: PropTypes.bool,
+  /**
+   * Override that shows a known "No Data" display. Used when there is known to be no value for a given clinical result concept at a specific datetime.
+   */
+  hasResultNoData: PropTypes.bool,
+};
+
+const defaultProps = {
+  resultData: {},
+  hideUnit: false,
+  isTruncated: false,
+  hasResultError: false,
+  hasResultNoData: false,
+};
+
+const ClinicalResult = (props) => {
+  const {
+    resultData,
+    hideUnit,
+    isTruncated,
+    hasResultError,
+    hasResultNoData,
+    ...customProps
+  } = props;
+
+  let clinicalResultDisplay = null;
+
+  if (hasResultError || hasResultNoData) {
```

Comment:
I'd split this up instad of checking hasResultError twice.
```jsx
  if (hasResultError) {
    <ResultError/>
  } else if (hasResultNoData) {
    <NoData />
  }
```

Reply:
Resolved here: https://github.com/cerner/terra-clinical/pull/574/commits/b32368e3d881a325105ce1a1d6ed262d0391bce0 , specifically [#diff-R74-R79](https://github.com/cerner/terra-clinical/pull/574/commits/b32368e3d881a325105ce1a1d6ed262d0391bce0#diff-a5d084114f980528ce4397f28581db7fR74-R79)